SOURCES/0008-feat-tests-add-unit-tests-concerning-ZFS-close-xcp-n.patch

```diff
@@ -141,8 +141,7 @@ index 0000000..c477fa3
 +            self.create_zfs_sr()
 +        except SR.SROSError as e:
 +            # Check SRUnavailable error.
-+            if e.errno != 47:
-+                raise
++            self.assertTrue(e.errno == 47)
```

Comment:
What's the reason for this change?

Reply:
Answer: this `if ` with a `raise` wasn't seen as a possible path thus coverage of the test files wasn't 100% and it failed.